media/starboard/starboard_renderer.cc

```diff
@@ -718,10 +721,26 @@ void StarboardRenderer::OnPlayerStatus(SbPlayerState state) {
 
 void StarboardRenderer::OnPlayerError(SbPlayerError error,
                                       const std::string& message) {
-  // TODO(b/375271948): Implement and verify error reporting.
   LOG(ERROR) << "StarboardRenderer::OnPlayerError() called with code " << error
              << " and message \"" << message << "\"";
-  NOTIMPLEMENTED();
+  switch (error) {
+    case kSbPlayerErrorDecode:
+      MEDIA_LOG(ERROR, media_log_) << message;
+      client_->OnError(PIPELINE_ERROR_DECODE);
+      break;
+    case kSbPlayerErrorCapabilityChanged:
+      MEDIA_LOG(ERROR, media_log_)
+          << (message.empty()
+                  ? kSbPlayerCapabilityChangedErrorMessage
+                  : base::StringPrintf("%s: %s",
+                                       kSbPlayerCapabilityChangedErrorMessage,
+                                       message.c_str()));
+      client_->OnError(PIPELINE_ERROR_DECODE);
+      break;
+    case kSbPlayerErrorMax:
+      NOTREACHED();
```

Comment:
Would this `kSbPlayerErrorMax` ever appear?

Reply:
I don't think so.